dev-tex/latex2html/latex2html-2022.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="The LaTeX2HTML translator"
+DESCRIPTION="* breaks up a document into one or more components as specified by the user
+* provides optional iconic navigation panels on every page which contain links to other parts of \
+the document
+* handles inlined equations, right-justified numbered equations, tables, or figures and any \
+arbitrary environment
+* can produce output suitable for browsers that support inlined images or character based \
+browsers (as specified by the user)
+* handles definitions of new commands, environments, and theorems even when these are defined in \
+external style files
+* handles footnotes, tables of contents, lists of figures and tables, bibliographies, and can \
+generate an Index
+* translates cross-references into hyperlinks and extends the LaTeX cross-referencing mechanism \
+to work not just within a document but between documents which may reside in remote locations
+* translates accent and special character commands to the equivalent HTML character codes where \
+possible
+* recognizes hypertext links (to multimedia resources or arbitrary internet services such as \
+sound/video/ftp/http/news) and links which invoke arbitrary program scripts, all expressed as \
+LaTeX commands
+* recognizes conditional text which is intended only for the hypertext version, or only for the \
+paper (PDF) version
+* can include raw HTML in a LaTeX document (e.g. in order to specify interactive forms)
+* can deal sensibly with all the commands and environments commonly used with LaTeX as summarized \
+at the back of the LaTeX blue book [1], and many of the packages described in the LaTeX \
+Companion, and others.
+* will try to translate any document with embedded LaTeX commands irrespective of whether it is \
+complete or syntactically legal."
+HOMEPAGE="https://www.latex2html.org/"
+COPYRIGHT="1995 Nikos Drakos
+	1997 Marek Rouchal
+	1996-1999 Jens Lippmann
+	1995-2002 Ross More"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/latex2html/latex2html/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9b3ba484226a2e39fb20695729370372b355f71bd65eeb4dd14f2c78699ed59a"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	latex2html$secondaryArchSuffix = $portVersion
```

Comment:
remove all references to $secondaryArchSuffix (not used without SECONDARY_ARCHITECTURES)